x-pack/plugins/security_solution/cypress/integration/overview/risky_hosts_panel.spec.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  OVERVIEW_RISKY_HOSTS_ENABLE_MODULE_BUTTON,
+  OVERVIEW_RISKY_HOSTS_LINKS,
+  OVERVIEW_RISKY_HOSTS_LINKS_ERROR_INNER_PANEL,
+  OVERVIEW_RISKY_HOSTS_LINKS_WARNING_INNER_PANEL,
+  OVERVIEW_RISKY_HOSTS_TOTAL_EVENT_COUNT,
+  OVERVIEW_RISKY_HOSTS_VIEW_DASHBOARD_BUTTON,
+} from '../../screens/overview';
+
+import { loginAndWaitForPage } from '../../tasks/login';
+import { OVERVIEW_URL } from '../../urls/navigation';
+import { cleanKibana } from '../../tasks/common';
+import { esArchiverLoad, esArchiverUnload } from '../../tasks/es_archiver';
+
+describe('Risky Hosts Link Panel', () => {
+  before(() => {
+    cleanKibana();
+  });
+
+  it('renders disabled panel view as expected', () => {
+    loginAndWaitForPage(OVERVIEW_URL);
+    cy.get(`${OVERVIEW_RISKY_HOSTS_LINKS} ${OVERVIEW_RISKY_HOSTS_LINKS_ERROR_INNER_PANEL}`).should(
+      'exist'
+    );
+    cy.get(`${OVERVIEW_RISKY_HOSTS_VIEW_DASHBOARD_BUTTON}`).should('be.disabled');
+    cy.get(`${OVERVIEW_RISKY_HOSTS_TOTAL_EVENT_COUNT}`).should('have.text', 'Showing: 0 hosts');
+    cy.get(`${OVERVIEW_RISKY_HOSTS_ENABLE_MODULE_BUTTON}`).should('exist');
+    cy.get(`${OVERVIEW_RISKY_HOSTS_ENABLE_MODULE_BUTTON}`)
+      .should('have.attr', 'href')
+      .and('match', /host-risk-score.md/);
+  });
+
+  describe('enabled module', () => {
+    before(() => {
+      esArchiverLoad('risky_hosts');
+    });
+
+    after(() => {
+      esArchiverUnload('risky_hosts');
+    });
+
+    it('renders disabled dashboard module as expected when there are no hosts in the selected time period', () => {
+      loginAndWaitForPage(
+        `${OVERVIEW_URL}?sourcerer=(timerange:(from:%272021-07-08T04:00:00.000Z%27,kind:absolute,to:%272021-07-09T03:59:59.999Z%27))`
+      );
+      cy.get(
+        `${OVERVIEW_RISKY_HOSTS_LINKS} ${OVERVIEW_RISKY_HOSTS_LINKS_WARNING_INNER_PANEL}`
+      ).should('exist');
+      cy.get(`${OVERVIEW_RISKY_HOSTS_VIEW_DASHBOARD_BUTTON}`).should('be.disabled');
+      cy.get(`${OVERVIEW_RISKY_HOSTS_TOTAL_EVENT_COUNT}`).should('have.text', 'Showing: 0 hosts');
+    });
+
+    it('renders dashboard module as expected when there are hosts in the selected time period', () => {
+      loginAndWaitForPage(OVERVIEW_URL);
+      cy.get(
+        `${OVERVIEW_RISKY_HOSTS_LINKS} ${OVERVIEW_RISKY_HOSTS_LINKS_WARNING_INNER_PANEL}`
+      ).should('not.exist');
+      cy.get(`${OVERVIEW_RISKY_HOSTS_VIEW_DASHBOARD_BUTTON}`).should('be.disabled');
```

Comment:
Should we add tests that cover cases when the dashboard button is enabled?

Reply:
there were some complexities around uploading dashboard saved objects from last time around, there is an issue to track that work here at elastic/security-team#1438. I will add that issue to our 7.16checklist as a backlog item